v3-nginx/nginx.conf

```diff
@@ -0,0 +1,33 @@
+load_module modules/ngx_http_modsecurity_module.so;
+
+user ${USER};
+worker_processes 1;
+
+error_log ${ERRORLOG} ${LOGLEVEL};
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections ${WORKER_CONNECTIONS};
+}
+
+http {
+
+    modsecurity on;
+    modsecurity_rules_file /etc/modsecurity.d/setup.conf;
+
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    log_format main '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log ${ACCESSLOG} main;
+    access_log ${METRICSLOG} combined;

```

Comment:
These lines may go into something like `/etc/nginx/conf.d/logging.conf`. What do you think?

Reply:
Not a bad idea